grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/tasks/internal/TasksManager.kt

```diff
@@ -62,9 +62,15 @@ internal class TaskManager @Inject constructor(
             grazelComponent
         )
 
+        val generateBuildifierScriptTask = GenerateBuildifierScriptTask.register(
+            rootProject
+        ) {
+
+        }
+
         // Root formatting task depends on sub project formatting and root generation task
         val formatBazelFilesTask = FormatBazelFileTask.register(rootProject) {
-            dependsOn(rootGenerateBazelScriptsTasks)
+            dependsOn(rootGenerateBazelScriptsTasks, generateBuildifierScriptTask)
```

Comment:
Please consider updating the task graph with the documentation as well 
https://grab.github.io/Grazel/gradle_tasks/#task-graph